pkgs/data/themes/pop-gtk/default.nix

```diff
@@ -9,17 +10,18 @@
 , gtk-engine-murrine
 , gdk-pixbuf
 , librsvg
+, python3
```

Comment:
Add `python3` to `nativeBuildInputs` and let `patchShebangs` fix the python scripts.

Reply:
I've applied the suggested modifications